flocker/acceptance/testtools.py

```diff
@@ -304,11 +304,15 @@ class Cluster(PRecord):
     :ivar Node control_node: The node running the ``flocker-control``
         service.
     :ivar list nodes: The ``Node`` s in this cluster.
-    :ivar treq: A ``treq`` client.
+
+    :ivar treq: A ``treq`` client, eventually to be completely replaced by
+        ``FlockerClient`` usage.
```

Comment:
Why eventually? Was there any particular reason for not going the whole way in this PR? 

I try to put judgement calls like this in the description of a PR.